app/Http/Controllers/MailController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Str;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+
+class MailController extends Controller
+{
+    public function claim(Request $request)
+    {
+        // Local id and token generation for now.
+        $email = $request->input('email');
+        $lockerid = Str::random(8);
```

Comment:
Although we do not really use GUID's, currently we're calling it GUID everywhere. Do not call this 'locker id', because a 'locker id' is something else.

Either wrongly use 'GUID' consistently or rename it to something else.

Reply:
Changed.